thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/override/OverrideConfigHelper.java

```diff
@@ -95,4 +101,54 @@ public static boolean isEnabled(Map<String, String> entityTargetLevel,
     targetEntity.put(EXCLUDED_FUNCTION_ID, functionIdString);
     return targetEntity;
   }
+
+  /**
+   * Convert a list of OverrideConfigDTO to a list of scaling factor, in which each scaling factor
+   * are filtered through target level.
+   *
+   * @param overrideConfigDTOs the list of OverrideConfigDTO
+   * @param timeSereisTargetLevel the
+   *                              filtration rule for applying OverrideConfigDTO
+   * @return a list of scaling factor
+   */
+  public static List<ScalingFactor> convertToScalingFactors(
+      List<OverrideConfigDTO> overrideConfigDTOs, Map<String, String> timeSereisTargetLevel) {
+    List<ScalingFactor> results = new ArrayList<>();
+    for (OverrideConfigDTO overrideConfigDTO : overrideConfigDTOs) {
+      if (OverrideConfigHelper.isEnabled(timeSereisTargetLevel, overrideConfigDTO)) {
+        long startTime = overrideConfigDTO.getStartTime();
+        long endTime = overrideConfigDTO.getEndTime();
+        double scalingFactor = 1.0d;
```

Comment:
This is just a suggestion: here and line 129 assume that we have a default scaling factor (start, end, 1.0d), but this is might not  be what we expect. If the config is not valid. we should expect Null and the out put result should be an empty list. If we do keep default 1.0d, we should add this to the log in line 126